config/dev.webpack.config.js

```diff
@@ -19,12 +19,20 @@ plugins.push(
             './RootApp': resolve(__dirname, '../src/DevEntry')
         },
         exclude: [ 'react-redux' ],
```

Comment:
You want to exclude the router package. Its already in the internal shared list and will not ve overriden.